python-sdk/dev/docker-compose.yaml

```diff
@@ -21,6 +21,8 @@ x-airflow-common:
     ASTRONOMER_ENVIRONMENT: local
     AIRFLOW__LINEAGE__BACKEND: openlineage.lineage_backend.OpenLineageBackend
     OPENLINEAGE_URL: http://host.docker.internal:5050/
+    OPENLINEAGE_NAMESPACE: "astro_dev"
+    OPENLINEAGE_EXTRACTORS: "astro_sdk.src.astro.extractors.extractor.PythonSDKExtractor"
```

Comment:
Will we need to register the extractor when we add Python SDK to runtime image?

Reply:
Yes that's what I understood from the documentation as well: https://docs.astronomer.io/astro/set-up-data-lineage

Reply:
@kaxil @sunank200 yes, we don't have a way to programatically register extractors. We assumed that there will be two types of extractors - ones in OpenLineage repo and custom extractors that live inside end-user's codebase. We did not think of shared extractors that live outside of OL's codebase.

However, maybe instead of adding another way to register extractors, we just should add `PythonSDKExtractor` to OpenLineage as "default" extractor that checks for existence of `get_openlineage_facets` method on operator and executes it if it exists.